spec/ttml1.xml

```diff
@@ -6654,9 +6654,6 @@ ref="semantics-style-inheritance-content"/> below.</p>
 other out-of-line style properties, thus creating a chain of references starting at the
 affected element. When expressed in this manner, the association of style information
 is referred to as <emph>chained referential styling</emph>.</p>
-<p>If the same style property is specified in more than one referenced
-style set, then the last referenced style set applies, where the order of application starts from
-the affected element and proceeds to referenced style sets, and, in turn, to subsequent referenced style sets.</p>
```

Comment:
I would still like to add a replacement sentence that explains that when chained referential styling is used each referenced style is fully resolved before the reference is followed.